client/components/tinymce/plugins/wpcom-track-paste/plugin.js

```diff
@@ -19,58 +23,35 @@ function trackPaste( editor ) {
 		return;
 	}
 
-	/**
-	 * Records the event.
-	 *
-	 * @param {String} mode - 'html-editor' or 'visual-editor', indicates which editor was in use on paste.
-	 * @param {(Array|DOMStringList)} types - The types the content is available to paste.
-	 */
-	function recordPasteEvent( mode, types ) {
-		debug( 'track paste event ' );
-		const typesStr = toString( types );
-		store.dispatch( recordTracksEvent( 'calypso_editor_content_paste', {
-			mode,
-			types: typesStr,
-			source: getSource( typesStr )
-		} ) );
-	}
+	const isGoogleDocsType = type =>
+		( type === 'application/x-vnd.google-docs-image-clip+wrapped' ) ||
+		( type === 'application/x-vnd.google-docs-document-slice-clip+wrapped' );
 
-	function isGoogleDocs( types ) {
-		return ( types.indexOf( 'application/x-vnd.google-docs-image-clip+wrapped' ) > -1 ) ||
-		( types.indexOf( 'application/x-vnd.google-docs-document-slice-clip+wrapped' ) > -1 );
-	}
+	const isGoogleDocs = types => types.filter( isGoogleDocsType ).length > 1;
```

Comment:
You're right, I misread the specification table for `some` and thought it was `es2015`. We seem to support IE11 and edge, so I'll go ahead and update the code to use `some` instead.